plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -50,5 +59,88 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 			</div>
 		);
 	}
-	return <div className="woocommerce-attribute-field"></div>;
+
+	const sortedAttributes = value.sort( ( a, b ) => a.position - b.position );
+	const attributeKeyValues = value.reduce(
+		(
+			keyValue: Record< number, ProductAttribute >,
+			attribute: ProductAttribute
+		) => {
+			keyValue[ attribute.id ] = attribute;
+			return keyValue;
+		},
+		{} as Record< number, ProductAttribute >
+	);
+	return (
+		<div className="woocommerce-attribute-field">
+			<Sortable
+				onOrderChange={ ( items ) => {
+					const newAttributes: ProductAttribute[] = items
+						.map( ( item, index ): ProductAttribute | undefined => {
+							const key = item.key
+								? parseInt( item.key as string, 10 )
+								: NaN;
+							if ( key !== NaN ) {
+								return {
+									...attributeKeyValues[ key ],
+									position: index,
+								};
+							}
+							return undefined;
+						} )
+						.filter(
+							( attr ): attr is ProductAttribute =>
+								attr !== undefined
+						);
+					onChange( newAttributes );
+				} }
+			>
+				{ sortedAttributes.map( ( attribute ) => (
+					<ListItem key={ attribute.id }>
+						<div>{ attribute.name }</div>
+						<div className="woocommerce-attribute-field__attribute-options">
+							{ attribute.options
+								.slice( 0, 2 )
+								.map( ( option, index ) => (
+									<div
+										className="woocommerce-attribute-field__attribute-option-chip"
+										key={ index }
+									>
+										{ option }
+									</div>
+								) ) }
+							{ attribute.options.length > 2 && (
+								<div className="woocommerce-attribute-field__attribute-option-chip">
+									+ { attribute.options.length - 2 }&nbsp;
+									{ __( 'more', 'woocommerce' ) }
```

Comment:
We might want to use `sprintf` here since not all languages will include "more" behind the number.